src/services/pps.ts

```diff
@@ -244,11 +244,11 @@ const pps = ({
         client.deletePipeline(
           deletePipelineRequest,
           credentialMetadata,
-          (error, res) => {
+          (error) => {
             if (error) {
               reject(error);
             }
-            return resolve(res.toObject());
+            return resolve({});
```

Comment:
Could we resolve without the empty object?

Reply:
We do `resolve({})` all over the place  - for some reason `resolve` expects `Empty.AsObject` and not undefined.